contrib/cmake/vcpkg_extra_triplets/x64-windows-mega.cmake

```diff
@@ -1,4 +1,12 @@
-set(VCPKG_VISUAL_STUDIO_PATH "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional")
+if(EXISTS "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional")
+    set(VCPKG_VISUAL_STUDIO_PATH "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional")
+elseif(EXISTS "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community")
+    message(AUTHOR_WARNING "Microsoft Visual Studio 2017 Community will be used. Professional would be more appropriate.")
+    set(VCPKG_VISUAL_STUDIO_PATH "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community")
+else()
+    message(FATAL_ERROR "Microsoft Visual Studio 2017 could not be found")
+endif()
+
```

Comment:
Should this be applied to `x64-windows-mega-staticdev.cmake` and `x86-windows-mega.cmake` too?

Reply:
Good idea, yes please

Reply:
done